.github/workflows/nmodl-ci.yml

```diff
@@ -100,7 +100,7 @@ jobs:
             ${{matrix.os}}-${{github.ref}}-
             ${{matrix.os}}-
 
-      - name: Build
+      - name: Build, Test and Install
```

Comment:
What problem is this solving? When I was debugging #741 (I had replaced the live debugging condition with `if: failure()`) then it was working for failures in the "Test" step.

Reply:
That's strange, because `failure()` only works for previous step and not steps. From my testing today, it didn't trigger.

Reply:
Weird. I had e.g. [this](https://github.com/BlueBrain/nmodl/runs/3731763593) live debug session working with [this](https://github.com/BlueBrain/nmodl/blob/33b451e410d14a2c77e1e1a6aef1de79a98d4d71/.github/workflows/nmodl-ci.yml) configuration.

Reply:
What do you want to do about this @alexsavulescu?

Reply:
Let's do one step, no added benefit in having three.

Reply:
OK. I just pushed a fix for the merge conflict.